tests/common/plugins/conditional_mark/issue.py

```diff
@@ -69,8 +70,29 @@ def is_active(self):
         Returns:
             bool: False if the issue is closed else True.
         """
+        global g_inv_name
+        INV_ENV_FILE = '../../../../ansible/group_vars/{}/env.yml'.format(g_inv_name)
+        PUBLIC_ENV_FILE = '../../../../ansible/group_vars/all/env.yml'
+        base_path = os.path.dirname(__file__)
+        env_file_path = os.path.join(base_path, INV_ENV_FILE)
+        if is_file_empty_or_comments(env_file_path):
+            env_file_path = os.path.join(base_path, PUBLIC_ENV_FILE)
+
+        try:
+            with open(env_file_path) as env_file:
+                proxy_env = yaml.safe_load(env_file)
+                if proxy_env is not None:
+                    proxy = proxy_env.get("proxy_env", {})
+                    http_proxy = proxy.get('http_proxy', '')
+                    https_proxy = proxy.get('https_proxy', '')
+                    proxies = {'http_proxy': http_proxy, 'https_proxy': https_proxy}
+                else:
+                    proxies = {'http_proxy': '', 'https_proxy': ''}
```

Comment:
The key in `proxies` should be `http` and `https` according to the python requests documentation.

Reply:
modified